package.json

```diff
@@ -178,7 +178,7 @@
     "optimize-css-assets-webpack-plugin": "^6.0.1",
     "prettier": "^2.7.1",
     "react-refresh": "^0.14.0",
-    "react-test-renderer": "^17.0.2",
+    "react-test-renderer": "18.2.0",
```

Comment:
Oppdaterer for å støtte react 18. Dette fikser de testene som feilet @tnarland 😇